### ISSUE TITLE
Setting tolerances with the ompl.app SE3RigidBodyPlanning Class 

### ISSUE THREAD

**Issue by IEI245MU5** [ompl/ompl#1041]
Hello, 

I am using the `SE3RigidBodyPlanning` Class to write a simple path planning function. However I have the issue that the meshes I am using are slightly clipping into each other at the starting position, leading to an `Error:   PRM: There are no valid initial states!` . Is there a way to increase tolerances, so that slight clipping is tolerated? Using the `setStateValidityCheckingResolution()` function does not work, regardless of the resolution i choose.

Here is my function:

```C++
bool plan(std::string robot_cad_path, 
                std::string env_cad_path, 
                int scale, 
                std::string solutionPath){

    // plan in SE3
    app::SE3RigidBodyPlanning setup;
 
    // load the robot and the environment
    setup.setRobotMesh(robot_cad_path);
    setup.setEnvironmentMesh(env_cad_path);
    
    // increase bounds to be larger than set environment
    setup.setBoundsAddition(2*scale);
 
    // define start state
    base::ScopedState<base::SE3StateSpace> start(setup.getSpaceInformation());
    start = setup.getDefaultStartState();
 
    // define goal state
    base::ScopedState<base::SE3StateSpace> goal(start);
    goal->setX(2*scale);
    goal->setY(2*scale);
    goal->setZ(2*scale);
    goal->rotation().setIdentity();
 
    // set the start & goal states
    setup.setStartAndGoalStates(start, goal);
 
    // setting collision checking resolution to 1% of the space extent
    setup.getSpaceInformation()->setStateValidityCheckingResolution(0.01);

    // select Planner
    setup.setPlanner(std::make_shared<geometric::PRM>(setup.getSpaceInformation()));
 
    // we call setup just so print() can show more information
    setup.setup();
 
    // try to solve the problem
    if (setup.solve(10))
    {
        // simplify the solution
        setup.simplifySolution();

        // save the solution path to a text file
        mkdir("../solutions", 0777);
        std::ofstream pathFile = std::ofstream(solutionPath + "/path.txt");
        setup.getSolutionPath().printAsMatrix(pathFile);
        pathFile.close();

        return true;
    }

    return false;
}
```

Any hints/tips are greatly appreciated. Thank you!

**Comment by zkingston** [comment]
Unfortunately no, there is no way to reduce tolerances on collision checking in OMPL.app. You may want to look into:
- Easy: sampling around the initial point until you find a valid starting configuration nearby.
- Hard: extending the [`FCLMethodWrapper`](https://ompl.kavrakilab.org/classompl_1_1app_1_1FCLMethodWrapper.html) to do what you want, I'm not sure if this is possible.

**Comment by IEI245MU5** [comment]
Thank you for your answer! 
I think my issue with the `Error:   PRM: There are no valid initial states! `was caused by setting the wrong start state. I wanted to have the starting position be the inherent position of the mesh from file. Therefore I was using `start = setup.getDefaultStartState();`, which causes the error. Because when I manually set the starting coordinates to where I know they should be, the program works.

So my question is: is there a way to have ompl.app set the start state space to the inherent position of the mesh?
